src/lwc/geFormRenderer/geFormRenderer.js

```diff
@@ -877,4 +890,33 @@ export default class GeFormRenderer extends NavigationMixin(LightningElement) {
         this.setFormFieldValue(donationImported, undefined, undefined);
         this.setFormFieldValue(paymentImported, undefined, undefined);
     }
+
+    /**
+     * @description Function that sets batch defaults on the BGE Form
+     * @param templateSections
+     * @returns {sections}
+     */
+    setBatchDefaults(templateSections) {
+        let sections = deepClone(templateSections);
+        if (isNotEmpty(this._batchDefaults)) {
+            let batchDefaultsObject = JSON.parse(this._batchDefaults);
+            sections.forEach(section => {
+                const elements = section.elements;
+                elements.forEach(element => {
+                    for (let key in batchDefaultsObject) {
+                        if (batchDefaultsObject.hasOwnProperty(key)) {
+                            const batchDefault = batchDefaultsObject[key];
+                            if (batchDefault.objectApiName === element.objectApiName &&
+                                batchDefault.fieldApiName === element.fieldApiName) {
+                                if (!isUndefined(batchDefault.value)) {
+                                    element.defaultValue = batchDefault.value;
```

Comment:
Do we need to worry about setting the display value as a default for lookups in addition to the field value? Or does that come along with the lookup behavior already?